tests/features/support/steps_clone.js

```diff
@@ -17,173 +18,173 @@ KEY:
 // const scope = require('./scope');
 
 // /* Of Note:
-// - We're using `*=` because sometimes da text has funny characters in it
-//     that are hard to anticipate, so we allow people to just put in some of the text
+// - We're using `*=` because sometimes da text has funny characters in it 
+//     that are hard to anticipate, so we allow people to just put in some of the text 
 // */
 
 
 // // =========================
 // // Contstants
 // // =========================
 // let ordinal_to_integer = {
-//   first: 0, second: 1, third: 2, fourth: 3, fifth: 4,
+//   first: 0, second: 1, third: 2, fourth: 3, fifth: 4, 
 //   sixth: 5, seventh: 6, eighth: 7, ninth: 8, tenth: 9,
 //   '1st': 0, '2nd': 1, '3rd': 2, '4th': 3, '5th': 4,
 //   '6th': 5, '7th': 6, '8th': 7, '9th': 8,'10th': 9,
 // };
 // let specified = '?"?(first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth|"[^"]+)?"?';
-// let click_with = { mobile: 'tap', pc: 'click', };
+// let click_with = { mobile: 'tap', pc: 'click', }; 
 // // non-capturing capturing group expression for single or double quotes?
 
 
 // =========================
 // Establishing
 // =========================
-//x @step(r'I spend at least ([0-9]+) seconds? on each page')
+//x @step(r'I spend at least ([0-9]+) seconds? on each page') 
 //x @step(r'I want to store screenshots in the folder "([^"]+)"')
 
 
 // =========================
 // Start
 // =========================
-//x @step(r'I start the possibly error-producing interview "([^"]+)"')
+//x @step(r'I start the possibly error-producing interview "([^"]+)"') 
 //x @step(r'I am using the server "([^"]+)"')
 //x @step(r'I log in with "([^"]+)" and "([^"]+)"')
-//x @step(r'I launch the interview "([^"]+)"')
+//x @step(r'I launch the interview "([^"]+)"') 
 
 // ? @step(r'I start the interview "([^"]+)"')
 // ? I start the interview file "([^"]+)" on mobile
-// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???)
+// ? I open the interview file ?"?([^"]+)"?( on mobile/a phone/a computer/a pc/a tablet???) 
 // ? I open the interview file ?"?([^"]+)"?[ on ]?(.*)/
-// √ I start the interview on mobile|pc
+// √ I start the interview on mobile|pc 
 
 
 // =========================
 // Observational/Passive
 // =========================
-//x @step(r'I wait forever')
+//x @step(r'I wait forever') 
```

Comment:
Is this equivalent to "I end the test"/"I end my interactions with the interview" or are we waiting for something to happen?

Reply:
I'm not sure. Maybe another place to check the code, but I think it just does what it says. Shall we leave it out?

Reply:
Leaving it out works for me.